cmd/presets/presets.go

```diff
@@ -508,8 +511,11 @@ networks:
 		".dockerignore": `/node_modules
 /vendor
 `,
-		"preset_language": "php",
-		"preset_create":   "kool docker kooldev/php:7.4 composer create-project --prefer-dist symfony/website-skeleton",
+		"preset_language":         "php",
+		"preset_create":           "kool docker kooldev/php:7.4 composer create-project --prefer-dist symfony/website-skeleton",
+		"preset_ask_services":     "database,cache",
+		"preset_database_options": "MySQL 8.0,MySQL 5.7,ProstgreSQL 13.0,none",
```

Comment:
Also for a next iteration, I would love to change this somehow into an YML config structure, for better structure.